### PR TITLE
Placeholder app ID & more detailed README instructions to get started

### PR DESCRIPTION
**Pull request by jznadams**

### CODE REVIEW

File: .idea/.name [hunk 1]
```diff
@@ -1 +1 @@
-sample-respoke-android
+sample-respoke-android
```
Comment (erinishimoticha) [inline .idea/.name:1]: Is your editor removing EOF newlines? Tisk, tisk, that's not POSIX compliant! :)

Comment (jznadams) [inline .idea/.name:1]: Android Studio is doing these things, not I. The new version changed a fair bit in these config files.

**Comment by robertcrabtree** [comment]
Looks good to me

File: .idea/.name (hunk 1 above)
Comment (chadxz) [inline .idea/.name:1]: http://stackoverflow.com/a/28660298/355743

**Pull request merged**